src/transport_ofi.c

```diff
@@ -1063,10 +1064,18 @@ int populate_av(void)
 
     for (i = 0; i < shmem_internal_num_pes; i++) {
         char *addr_ptr = alladdrs + i * shmem_transport_ofi_addrlen;
-        shmem_runtime_get(i, "fi_epname", addr_ptr, shmem_transport_ofi_addrlen);
+        int err = shmem_runtime_get(i, "fi_epname", addr_ptr, shmem_transport_ofi_addrlen); //TODO Ask Jim about having this check implemented
+        if(err != 0){
+            RAISE_WARN_STR("Runtime get failed");
```

Comment:
I think this should be treated as a fatal error rather than a warning, using `RAISE_ERROR_STR`.  And same below.